services-directions-models/src/test/java/com/mapbox/api/directions/v5/utils/MutateJsonUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.mapbox.api.directions.v5.utils;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+
+import java.util.Map;
+
+public class MutateJsonUtil {
+
+  public static void mutateJson(JsonElement element) {
```

Comment:
Could you add a short comment on what's the goal for this chunk of code? It's not too hard to follow but would be good to have an overview for future refactors.

Reply:
Added